apps/nuxt3-ssr/Dockerfile

```diff
@@ -42,6 +42,9 @@ ENV NUXT_PUBLIC_COHORT_ONLY=$NUXT_PUBLIC_COHORT_ONLY
 # pass optional analytics key 
 ENV NUXT_PUBLIC_ANALYTICS_KEY=$NUXT_PUBLIC_ANALYTICS_KEY
 
+# pass optional name of analytics provider (for example: "google" defaults to "siteimprove") 
```

Comment:
sorry, I changed this but now I am confused, should it be 'google' or 'google-analytics'. Make same everywhere?